### PR TITLE
BITMAKER-2092: Implement Profile Settings mockup

### PR DESCRIPTION
**Pull request by erick-GeGe**
# Description

estela-web: Implement "Project Settings" mockup

# Issue

https://tasks.bitmaker.dev/issues/2092

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] My code follows the style guidelines of this project.
- [ ] I have made corresponding changes to the [documentation](https://github.com/bitmakerla/estela/tree/main/docs).
- [ ] New and existing tests pass locally with my changes.
- [ ] If this change is a core feature, I have added thorough tests.
- [ ] If this change affects or depends on the behavior of other _estela_ repositories, I have created pull requests with the relevant changes in the affected repositories. Please, refer to our [official documentation](https://estela.bitmaker.la/docs/).
- [ ] I understand that my pull request may be closed if it becomes obvious or I did not perform all of the steps above.

### CODE REVIEW

File: estela-web/src/components/SettingsPasswordPage/styles.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+// .create-new-project {
+//     margin: 30px auto;
+//     display: flex;
+//     align-self: auto;
+// }
+
+// .container {
+//     background-color: #FBFCFD;
+// }
```
Comment (raymond1242) [inline estela-web/src/components/SettingsPasswordPage/styles.scss:35]: Remove comments

File: installation/Makefile [hunk 2]
```diff
@@ -26,7 +26,7 @@ resources:
 	-$(DOCKER_COMPOSE) -f local/docker-compose.yaml up ${RESOURCES} -d
 	-minikube delete
 	-. ./local/.env && minikube start \
-	           --feature-gates="TTLAfterFinished=true" \
+		   --feature-gates="TTLAfterFinished=true" \
```
Comment (raymond1242) [inline installation/Makefile:29]: Is there an issues in this line ?

File: installation/Makefile [hunk 3]
```diff
@@ -11,7 +11,7 @@ ifeq ($(OS),Windows_NT)
 else
 	UNAME_S := $(shell uname -s)
 	ifeq ($(UNAME_S),Linux)
-		DOCKER_COMPOSE = docker-compose
+		DOCKER_COMPOSE = docker compose
```
Comment (raymond1242) [inline installation/Makefile:14]: Please confirm this is Ok in linux?

File: installation/Makefile (hunk 2 above)
Comment (erick-GeGe) [reply installation/Makefile]: Yes, it gave me errors in Linux-Ubuntu

File: installation/Makefile (hunk 3 above)
Comment (erick-GeGe) [reply installation/Makefile]: using docker-compose gives me errors, because the version that I install uses docker compose

**Comment by erick-GeGe** [comment]
problems solved

File: estela-web/src/components/SettingsDataPersistencePage/styles.scss [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+.ant-radio-button-wrapper:hover {
+    color: $estela !important;
+    background-color: $estela-blue-low;
+}
+
+.ant-radio-button-wrapper-checked{
+    border: 1px solid $estela !important;
+    background-color: $estela-blue-low;
+    color: $estela !important;
+}
```
Comment (raymond1242) [inline estela-web/src/components/SettingsDataPersistencePage/styles.scss:50]: Add empty line of the file

File: estela-web/src/components/SettingsPasswordPage/styles.scss [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+.btn_sendRequest{
+    background-color: #9BA2A8;
+    border-radius: 8px;
+    color: white;
+    padding: 28px 15px;
+    line-height: 0px;
+    width: 100%;
+    font-weight: 500;
+    font-size: 16px;
+}
```
Comment (raymond1242) [inline estela-web/src/components/SettingsPasswordPage/styles.scss:21]: Add new line at the end of the file

File: installation/Makefile [hunk 6]
```diff
@@ -26,7 +26,7 @@ resources:
 	-$(DOCKER_COMPOSE) -f local/docker-compose.yaml up ${RESOURCES} -d
 	-minikube delete
 	-. ./local/.env && minikube start \
-	           --feature-gates="TTLAfterFinished=true" \
+			   --feature-gates="TTLAfterFinished=true" \
 		   --insecure-registry=$${HOST_REGISTRY} \
 		   --cpus="2" \
                    --memory="2500mb" \
```
Comment (raymond1242) [inline installation/Makefile:32]: Everything about the makefile don't upload in the PR, because is own your configuration

**Pull request closed without merging**